code/src/components/CustomCheckbox.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
```

Comment:
wow! is this the props we can use for styling? like, how, what, how do you do this? I'm impressed!